src/lib.rs

```diff
@@ -29,12 +34,23 @@
 //! ```no_run
 //! # extern crate tokio;
 //! # extern crate fantoccini;
-//! use fantoccini::{Client, Locator};
+//! # #[cfg(feature = "rustls-tls")]
```

Comment:
This example now won't compile if no feature are enabled (which is also a valid way to compile the crate). I suggest that in these examples we just use the `openssl-tls` version, put
```rust
#![cfg(feature = "openssl-tls")]
```
at the top of the example (so it'll be ignored when the feature isn't enabled), and the explain in the documentation to see a later section of the docs for details on how to use a different TLS implementation.